pom.xml

```diff
@@ -1505,7 +1505,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-annotations</artifactId>
-                <version>2.8.11.1</version> <!-- TODO sync. with jackson.version -->
```

Comment:
Wait, it looks like we are still using Jackson 2.8.11?  Then don't change this.  jackson-annotations-2.8.11 is buggy.  There is no 2.8.11.1 version of the other Jackson artifacts.  Sorry, thought we had upgraded the rest of Jackson.